templates/virtualization-controller/_helpers.tpl

```diff
@@ -35,6 +35,16 @@
 {{- if (hasKey .Values.virtualization "virtualImages") }}
 - name: VIRTUAL_IMAGE_STORAGE_CLASS
   value: {{ .Values.virtualization.virtualImages.storageClassName }}
+- name: VIRTUAL_IMAGE_DEFAULT_STORAGE_CLASS
+  value: {{ .Values.virtualization.virtualImages.defaultStorageClassName }}
```

Comment:
I suggest refraining from using this option at the moment and waiting for the task to be completed, when this code will be removed along with the other. This will help prevent confusion.